src/main/resources/application.yml

```diff
@@ -57,6 +60,8 @@ spring:
       password:
    rest:
       iexBasePath: https://api.iextrading.com/1.0
+      iexHistoricalPath: https://cloud.iexapis.com/stable
+      iexApiToken: pk_fde37fe1b888413ab706e1aab237aebd
```

Comment:
In the real world, you would never put a private key in a file that you commit to git. Generally, this would go in a password.properties file that is registered in the .gitignore file so that it doesn't get committed.